### ISSUE TITLE
Backend: Install multer storage library

### ISSUE THREAD

**Issue by senner007** [HackYourFuture-CPH/simply-name.it#218]
**Description**

Install the multer storage library to store image files in a local folder:

**Comment by Midnighter** [comment]
Is the idea to serve that directory directly with nginx or whatever Heroku provides? If so, how is authentication handled in that case?

**Comment by senner007** [comment]
> Is the idea to serve that directory directly with nginx or whatever Heroku provides? If so, how is authentication handled in that case?

Yes, directly but still using the auth middleware. The frontend will have to use ajax to send headers also
